archetypes/mp-bare/src/main/resources/archetype-resources/src/test/resources/META-INF/microprofile-config.properties.vm

```diff
@@ -0,0 +1,4 @@
+# Override configuration in main source branch, so we do not use 8080 port for tests
```

Comment:
I don't understand this comment.

Reply:
That was also copied from the quickstart, I think this is only saying override the default config to use a random port.

Reply:
I've updated the comment to `# Override configuration to use a random port for the unit tests`